torch/csrc/jit/codegen/cuda/type.h

```diff
@@ -152,7 +152,16 @@ enum class UnaryOpType {
   Trunc,
 
   // Might be a bitwise operator or boolean operator.
-  Not
+  Not,
+
+  // Memory ops,
+  //  TODO & Note: Should consider moving these into
+  // a separate/new IR node, maybe LDST? Also should be moved
```

Comment:
Yes, I agree. I think it makes sense to have a separate class for each of these ops, rather than having a single class for all types of loads and stores.

Reply:
Added a `LoadStore` IR node to keep track of the new memory ops. Could consider adding more once we are adding more logic into it.